ansible/roles/commcarehq/tasks/main.yml

```diff
@@ -105,3 +105,16 @@
   tags:
     - localsettings
     - hq-localsettings
+
+- include_vars: "../../../../environments/{{ deploy_env }}/app-processes.yml"
```

Comment:
I see why this was the easiest way to do this; at the same time I can see a couple drawbacks at least long-term. One is that this only works for our envs, but not if someone's passing in `COMMCARE_CLOUD_ENVIRONMENTS` to use their own envs dir. The other is that if one of these templates relies on `environmental-defaults/app-processes.yml`, that won't get included here.

I think the longer-term solution is for `commcare-cloud` to drop a freshly generated `.combined-vars.yml` file in the env right before calling ansible, and include that file in addition to (or instead of) `public.yml`. Taking that extra step will will give us total control over how ansible variables get set and decouples that from the config format, letting us define configs based on what's the most sensical for the maintainer of the environment to write rather than what ansible needs in order to be able to run.

I think if this works (i.e. if none of this requires the environmental-defaults) then I'm happy to merge this in now, and then I'll implement that that variables change in a separate PR.

Reply:
Totally agree this isn't correct long term. I cringed a bit when figuring out the correct number of `../`, but wanted to get something out there so you guys could review/we could come to consensus

Neither the pillowtop nor the form feed supervisor require that defaults file, so this should be fine for this PR.

It sounds like you have a much better grasp of where you're taking this, so I think that I'll leave that for you to do.